src/MoBi.UI/Views/DescriptorConditionListView.cs

```diff
@@ -53,12 +54,19 @@ public override void InitializeBinding()
             .AsReadOnly();
 
          _gridViewBinder.Bind(dto => dto.Tag)
-            .OnValueUpdating += (o, e) => OnEvent(() => onCritiraTagChanged(o, e));
+            .OnValueUpdating += (o, e) => OnEvent(() => onCriteriaTagChanged(o, e));
+
+         _gridViewBinder.AddUnboundColumn()
```

Comment:
I always wanted to add this guy to delete a condition instead of having to right click on it. So cumbersome!